packages/upload-box/__tests__/index.test.js

```diff
@@ -0,0 +1,45 @@
+// import React from 'react'
+// import renderer from 'react-test-renderer'
+// import { ThemeProvider } from '@hixme-ui/theme'
+import 'jest-styled-components'
+
+import UploadBox from '../src/index'
+
+describe('UploadBox', () => {
+  // test('Should render okay', () => {
+  //   const component = renderer.create(
+  //     <ThemeProvider>
+  //       <UploadBox />
+  //     </ThemeProvider>,
+  //   )
+
+  //   const tree = component.toJSON()
+  //   expect(tree).toBeDefined()
+  // })
```

Comment:
This test fails. I'm not sure why.

Reply:
Dropzone is looking to bind events to the window object.  Signs appear to point to JSDOM and some jest global setup.

Reply:
Thanks. I'll check it out.